catalog/templates/catalog/bookinstance_list_borrowed_user.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base_generic.html" %}
+
+{% block content %}
+<h1>Borrowed books</h1>
+
+{% if bookinstance_list %}
+<ul>
+
+  {% for bookinst in bookinstance_list %}
+  <li class="{% if bookinst.is_overdue %}text-danger{% endif %}">
+    <a href="{% url 'book-detail' bookinst.book.pk %}">{{ bookinst.book.title }}</a> ({{ bookinst.due_back }})
+  </li>
+  {% endfor %}
+</ul>
+
+{% else %}
+<p>There are no books borrowed.</p>
+{% endif %}
+{% endblock %}
```

Comment:
Phải thụt tab để phân cấp trong các block code e nhé
```suggestion
{% block content %}
  <h1>Borrowed books</h1>

  {% if bookinstance_list %}
    <ul>
      {% for bookinst in bookinstance_list %}
        <li class="{% if bookinst.is_overdue %}text-danger{% endif %}">
          <a href="{% url 'book-detail' bookinst.book.pk %}">{{ bookinst.book.title }}</a> ({{ bookinst.due_back }})
        </li>
      {% endfor %}
    </ul>
  {% else %}
    <p>There are no books borrowed.</p>
  {% endif %}
{% endblock %}
```